css-basics/css-basics_combined-users/css/styles.css

```diff
@@ -46,23 +46,47 @@ li {
   background-color: white;
 }
 
+ul .user .bio {
+  visibility: hidden;
+}
+ul .user .location {
+  text-align: left;
+  color: darkslategray;
+}
+
+aside .user {
+  max-width: 250px;
+  padding: 12px;
+  border-radius: 4px;
+}
 .name {
   font-size: 18px;
   text-transform: uppercase;
   margin: 0;
 }
 
+aside .name {
+  font-size: 16px;
+}
 .bio {
   font-size: 14px;
   color: darkslategray;
 }
 
+aside .bio {
```

Comment:
`aside .user .bio`? Maybe it doesn't make any difference here.

Reply:
🤷‍♀️ same